### PR TITLE
Make more consistent and remove redundant comma

### PR DESCRIPTION
**Pull request by HR**
In order to prevent confusion

### CODE REVIEW

File: docs/tutorial/using-selenium-and-webdriver.md [hunk 1]
```diff
@@ -1,23 +1,14 @@
 # Using Selenium and WebDriver
-
```
Comment (zcbenz) [inline docs/tutorial/using-selenium-and-webdriver.md]: Space after title should be kept.

File: docs/tutorial/using-selenium-and-webdriver.md [hunk 2]
```diff
@@ -1,23 +1,14 @@
...
 From [ChromeDriver - WebDriver for Chrome][chrome-driver]:
 
-> WebDriver is an open source tool for automated testing of web apps across many
-> browsers. It provides capabilities for navigating to web pages, user input,
-> JavaScript execution, and more. ChromeDriver is a standalone server which
-> implements WebDriver's wire protocol for Chromium. It is being developed by
-> members of the Chromium and WebDriver teams.
+> WebDriver is an open source tool for automated testing of web apps across many browsers. It provides capabilities for navigating to web pages, user input, JavaScript execution, and more. ChromeDriver is a standalone server which implements WebDriver's wire protocol for Chromium. It is being developed by members of the Chromium and WebDriver teams.
```
Comment (zcbenz) [inline docs/tutorial/using-selenium-and-webdriver.md]: Lines should be wrapped with 80 columns.

**Comment by HR** [comment]
Fixed them. Fine now?

**Comment by zcbenz** [comment]
__All__ spaces after title should be kept, and __all__ lines should be wrapped with 80 columns.

**Comment by zcbenz** [comment]
And for trailing commas, they are actually preferred:
http://www.puppetcookbook.com/posts/trailing-commas.html

Sorry but this PR makes things inconsistent.

**Pull request closed without merging**